Client/Frontend/Browser/BrowserScrollController.swift

```diff
@@ -37,12 +37,14 @@ class BrowserScrollingController: NSObject {
     weak var snackBars: UIView?
 
     var footerBottomConstraint: Constraint?
-    var headerTopConstraint: Constraint?
+    // TODO: Since SnapKit hasn't added support yet (Swift 2.0/iOS 9) for handling layoutGuides,
+    // this constraint uses the system abstraction instead of SnapKit's Constraint class
+    var headerTopConstraint: NSLayoutConstraint?
```

Comment:
Oh snap - I didnt know that existed! Yes, we can use that but it looks like it's broken on SnapKit's swift-2.0 branch but there is a PR open for this here: https://github.com/SnapKit/SnapKit/pull/137. I'm going to file a follow up bug to change these back to Snapkit after that patch is merged in.